#!/usr/bin/env python
# field of view configuration

PACKAGE="rgcnn_models"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("z_lower_limit", double_t, 1, "Z Lower limit", 0.0, 0.0, 15.0)
gen.add("z_upper_limit", double_t, 1, "Z Upper limit", 3.0, 0.0, 50.0)

gen.add("x_lower_limit", double_t, 1, "X Lower limit", -0.5, -1.5, 0.0)
gen.add("x_upper_limit", double_t, 1, "X Upper limit",  0.50, 0.0, 1.5)

gen.add("y_lower_limit", double_t, 1, "Y Lower limit", -0.5, -1.5, 0.0)
gen.add("y_upper_limit", double_t, 1, "Y Upper limit", 0.50, 0.0, 1.5)

exit(gen.generate(PACKAGE, "rgcnn_models", "passthrough_filter_node"))
